event emitters tests in action_bar component

Nishant-Nagururu opened the pull request:
### Fixes #
This PR helps fix issue #1717 which is an ongoing issue that requests more tests for components in altair-app.

Checks

 Ran yarn test-build
 Updated relevant documentations
 Updated matching config options in altair-static

### Changes proposed in this pull request:
I added tests for the event_emitters for the action_bar component to make sure they go off properly.

imolorhe reviewed (changes requested):
The tests don't pass

Nishant-Nagururu commented:
I'm sorry @imolorhe that these did not work the first time. I am first time contributor. I looked closely at tests for other components and modeled mine after those. Please let me know if they still are incorrect.

imolorhe reviewed (approved):
Looks good. Thanks.

Nishant-Nagururu commented:
@imolorhe Is there anything else I need to do for this PR?

imolorhe reviewed (changes requested):
Sorry some tests are still failing https://github.com/altair-graphql/altair/actions/runs/7674181736/job/20919231287?pr=2411

Nishant-Nagururu commented:
@imolorhe sorry for the back and forth but the errors should be fixed now. I updated the snapshot to reflect the classes I added in the html and I moved a class that I accidentally put in an inaccurate spot. The two failing tests should be resolved now.
